src/timeline-list/index.tsx

```diff
@@ -73,6 +75,12 @@ const TimelineList = (props: TimelineListProps) => {
     prevDate.current = date;
   }, [updateSource]);
 
+  const shouldApplyRTL = useMemo(() => constants.isRTL && constants.isAndroid, []);
```

Comment:
```suggestion
  const shouldUseAndroidRTLFix = useMemo(() => constants.isRTL && constants.isAndroid, []);
```
Just to keep unified naming, or better let's move this to the 'commons/constants' file